### ISSUE TITLE
Check autocomplete combo box for a keyboard trap bug

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#3262]
Testing of Notfiy's autocomplete combo box uncovered an edge case bug. Let's check our implementation to make sure the issue is resolved:

Steps to reproduce:
So this seems to happen in Firefox, and the steps to replicate are very precise. I’m not on voiceover, just doing keyboard navigation:

- Tab until the autocomplete box
- Escape
- Shift-tab to move focus upwards
- Shift once to focus the autocomplete box again
- The next shift will focus at the top of the page, and you will be trapped within the top and autocomplete box elements.

**Comment by connorscarolyns** [comment]
@muoyopopo This should have been tested during the fabel test. Did we see any evidence of a key trap when users used the typeahead feature?

**Comment by srtalbot** [comment]
@connorscarolyns can we retest to see if this is still an issue?

**Comment by connorscarolyns** [comment]
We did this in the last round of fable testing. No keyboard trap was found.
